README.md

```diff
@@ -10,7 +10,8 @@ A CLI tool to determine if an instance is PAYG or BYOS across different CSPs
 
 	Not valid instance metadata verified successfully
 
-**12 - Unknown**
+**12 - BYOS**
```

Comment:
shouldn't this change be in a different PR ?

Reply:
Will not squash merge so the commits are preserved.